src/scale-manager/SafeScaleManager.spec.js

```diff
@@ -45,6 +45,16 @@ describe('Scale Manager', () => {
     expect(sm.callback.callCount).to.equal(0);
   });
 
+  it('Should not call resize in IE 11 environemnt', () => {
+    sm.callback = Sinon.fake();
+
+    Sinon.replace(window, 'dispatchEvent', () => {});
+    sm.resizer.resize();
+    Sinon.restore();
```

Comment:
Move this `restore` call to an `afterEach`. That way, if this test begins to fail the missing `restore` call will always happen keeping other tests from failing..

Reply:
Fixed